src/Microsoft.Identity.Web.OidcFIC/PublicAPI/net462/InternalAPI.Unshipped.txt

```diff
@@ -1 +0,0 @@
-#nullable enable
```

Comment:
We need to keep the `#nullabe enable` (for future unshipped). Maybe we'd want to change the script?

Reply:
probably change the script, but we'll leave it for now.